Jenkinsfile

```diff
@@ -72,6 +72,9 @@ pipeline {
         string(name: 'TestTag',
                defaultValue: "",
                description: 'Test-tag to use for this run (i.e. pr, daily_regression, full_regression, etc.)')
+        string(name: 'BuildType',
+               defaultValue: "",
+               description: 'Type of build.  (I.e. dev or release).  Defaults to release on an RC or dev otherwise.')
```

Comment:
Minor: The beauty of this is it also allows us to use debug which adds -O0 and -g